laravel/resources/views/banners.blade.php

```diff
@@ -104,7 +110,9 @@
     @endif
 </div>
 @can('add banners')
-    @include('modals.banners.modal-add')
+    @if ($banners->count() == 0)
```

Comment:
[ERROR]

I believe, you wanted here this, don't you?
```suggestion
    @if ($instance_list->count() > 0)
```